.github/workflows/CI-notebook.yml

```diff
@@ -0,0 +1,36 @@
+name: CI Notebooks
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  ci-notebook:
+    strategy:
+      matrix:
+        operatingSystem: [ubuntu-latest, windows-latest]
+        pythonVersion: [3.6, 3.7, 3.8]
```

Comment:
I think you may need to limit this to 3.6, 3.7 since shap 0.34.0 which we depend on only supports 3.5-3.7.  We can't drop 3.5 support and move to latest shap because we are required to support it still.

Reply:
Who's required to support 3.5 with raiwidgets?

Reply:
If 3.8 really isn't supported we should remove it from setup.py. We're currently claiming that this works